packs/data/outlaws-of-alkenstar-bestiary.db/shoma-lyzerius.json

```
@@ -387,6 +387,126 @@
             },
             "type": "weapon"
         },
+        {
+            "_id": "ZQSgJYkU0FsNyj3m",
+            "flags": {
+                "core": {
+                    "sourceId": "Compendium.pf2e.equipment-srd.gWr4q4HiyGhETA8H"
+                }
+            },
+            "img": "icons/consumables/potions/potion-tube-corked-green.webp",
+            "name": "Lyzerium (Moderate)",
+            "sort": 400000,
+            "system": {
+                "MAP": {
+                    "value": ""
+                },
+                "baseItem": "alchemical-bomb",
+                "bonus": {
+                    "value": 1
+                },
+                "bonusDamage": {
+                    "value": 0
+                },
+                "category": "martial",
+                "containerId": null,
+                "damage": {
+                    "damageType": "fire",
+                    "dice": 2,
+                    "die": "d8",
+                    "persistent": {
+                        "faces": null,
+                        "number": 2,
+                        "type": "fire"
+                    }
+                },
+                "description": {
+                    "value": "<p><strong>Activate</strong> <span class=\"pf2-icon\">A</span> Strike</p>\n<p>A bottle of lyzerium is alchemist’s fire, except it has negligible bulk, explodes in a green flame, and is worth twice as much.</p>\n<p>Alchemist's fire is a combination of volatile liquids that ignite when exposed to air. Alchemist's fire deals 2d8 fire damage, 2 persistent fire damage, and 2 fire splash damage. You gain a +1 item bonus to attack rolls.</p>"
```

```suggestion
                    "value": "<p><strong>Activate</strong> <span class=\"pf2-icon\">1</span> Strike</p>\n<p>A bottle of lyzerium is alchemist’s fire, except it has negligible bulk, explodes in a green flame, and is worth twice as much.</p>\n<p>Alchemist's fire is a combination of volatile liquids that ignite when exposed to air. Alchemist's fire deals 2d8 fire damage, 2 persistent fire damage, and 2 fire splash damage. You gain a +1 item bonus to attack rolls.</p>"
```

Cause I'm opinionated